### PR TITLE
expanded-guideline-ods-5

### PR DESCRIPTION
**Pull request by Asbjoedt**

### CODE REVIEW

**Review by carlwilson — COMMENTED** [review]
I'm still uncomfortable about the talk of removal without any consideration of the technical process of removal. It's not as simple as deleting a file and feels like an encouragement to take preservation risks.

**Comment by carlwilson** [comment]
Again, we're conflicted here @Asbjoedt

**Comment by Asbjoedt** [comment]
If there's no appropriate automatic solutions for this, it may be a manual handling of the document by opening it in an application and replacing or removing the embedded files. This should not result in any technical malformities of the spreadsheet. Regarding the automatic solution, we cannot expect any automatic solution to be created by the market or any organisation unless the specification mandating this approach exists. First spec, then tools.

Discussing automatic removal of data from spreadsheets is something you have created an issue on, and we may keep the disucssion there IMO.

I resolved the conflict.

**Review by RvanVeenendaal — CHANGES_REQUESTED** [review]

File: Draft v1.0/Specification.md [hunk 1]
```diff
@@ -183,7 +183,7 @@ The following table gives guidelines for how to comply with the requirements in
...
-| ODS_5 | Embedded objects can be converted to a file format, your organisation accepts. Secondly, embedded objects can be removed with an appraised acceptance of the data loss. |
+| ODS_5 | Embedded objects are allowed in the ODF standard, and you may encounter these. Typically, the embedded object is an image file and very often so a logo image, but embedded objects may also be other spreadsheets, text documents, audio, video and even 3D objects.<br><br>If the embedded object has a file format, your organisation accepts, this policy will validate succesfully, but if not the embedded object should be converted to a file format, your organisation accepts. Secondly, embedded objects can be removed from the spreadsheet and either preserved as indenpendent objects or deleted entirely with an appraised acceptance of the data loss.<br><br>However, if you decide to remove any embedded objects from the spreadsheet, you should document the object and its relationship to the spreadsheet. This metadata can be saved in a sidecar file or associated table. The metadata must include the cell range which the embedded object covered. |
```
[the remark this answers is not in the document]
Comment (Asbjoedt) [reply Draft v1.0/Specification.md]: Kaido and Asbjørn looked at your proposed text and we think it is excellent and have adopted it in the pull request.

**Review by kivilaan — APPROVED** [review]

**Review by RvanVeenendaal — APPROVED** [review]

**Pull request merged**